mappings/net/minecraft/entity/data/DataTracker.mapping

```diff
@@ -1 +1,30 @@
 CLASS net/minecraft/class_216 net/minecraft/entity/data/DataTracker
+	FIELD field_1740 DATA_TYPE_MAP Ljava/util/HashMap;
+	FIELD field_1741 trackedData Ljava/util/Map;
+	FIELD field_1742 dirty Z
+	METHOD method_1500 isDirty ()Z
+	METHOD method_1501 getByte (I)B
+		ARG 1 id
+	METHOD method_1502 startTracking (ILjava/lang/Object;)V
+		ARG 1 key
+		ARG 2 value
+	METHOD method_1503 readEntries (Ljava/io/DataInputStream;)Ljava/util/List;
+		ARG 0 input
+	METHOD method_1504 writeAllEntries (Ljava/io/DataOutputStream;)V
+		ARG 1 output
+	METHOD method_1505 writeEntry (Ljava/io/DataOutputStream;Lnet/minecraft/class_270;)V
+		ARG 0 output
+		ARG 1 entry
+	METHOD method_1506 writeEntries (Ljava/util/List;Ljava/io/DataOutputStream;)V
+		ARG 0 entries
+		ARG 1 output
+	METHOD method_1507 getDirtyEntries ()Ljava/util/ArrayList;
+	METHOD method_1508 getInt (I)I
+		ARG 1 id
+	METHOD method_1509 set (ILjava/lang/Object;)V
+		ARG 1 id
+		ARG 2 object
+	METHOD method_1510 getString (I)Ljava/lang/String;
+		ARG 1 id
+	METHOD method_1511 updateEntries (Ljava/util/List;)V
```

Comment:
`writeUpdatedEntries` according to yarn - https://github.com/FabricMC/yarn/blob/4674b66184231baebe54ca8bf95568fd677f25f8/mappings/net/minecraft/entity/data/DataTracker.mapping#L14